src/utils/helper.ts

```diff
@@ -11,9 +11,7 @@ export const scrollToTop = (scrollContentId: string) =>
 
 export const orderCardIfNeeded = (cards: TSortableItems[], userCardConfig: string[] | undefined) =>
   userCardConfig
-    ? cards.sort((a, b) => {
-        return userCardConfig.indexOf(a.id) > userCardConfig.indexOf(b.id) ? 1 : -1;
-      })
+    ? cards.sort((a, b) => (userCardConfig.indexOf(a.id) > userCardConfig.indexOf(b.id) ? 1 : -1))
```

Comment:
Beware that `sort` modifies the original array. Are you sure that `cards` are not modify in the parent component?

Reply:
I did not change anything here, it's just the linter that reorganized the code

Reply:
Added ```slice()``` as discussed